packages/invoices/src/OutboundShipment/DetailView/modals/ItemDetailsForm.tsx

```diff
@@ -85,22 +79,20 @@ export const ItemDetailsForm: React.FC<ItemDetailsFormProps> = ({
       <ModalRow>
         <ModalLabel labelKey="label.item" />
         <Grid item flex={1}>
-          <ItemSearchInput value={selectedItem} onChange={onChangeItem} />
+          <ItemSearchInput
+            currentItemId={summaryItem?.itemId}
+            onChange={onChangeItem}
+          />
```

Comment:
have added an optional `itemId` to the item search input instead of passing an item if wanted